src/server/services/proxy/views/environment.njk

```diff
@@ -0,0 +1,153 @@
+{% extends "layouts/tabbed-page.njk" %}
+
+{% from "icons/pending-icon/macro.njk" import appPendingIcon %}
+{% from "icons/tick-icon/macro.njk" import appTickIcon %}
+{% from "info/macro.njk" import appInfo %}
+{% from "loader/macro.njk" import appLoader %}
+{% from "tag/macro.njk" import appTag %}
+{% from "tool-tip/macro.njk" import appToolTip %}
+
+{% set setupProxyHtml %}
+  To set up proxy, contact the Platform team via Slack
+  <a class="app-link app-link--underline app-link--text-colour" href="{{ supportChannel }}" target="_blank"
+     rel="noopener noreferrer">#cdp-support</a>
+{% endset %}
+
+{% set modifyProxyRulesHtml %}
+  To modify proxy rules,
+  follow the <a class="app-link app-link--underline app-link--text-colour" href="{{ proxyDocumentationUrl }}" target="_blank"
+     rel="noopener noreferrer">proxy documentation</a>.
+{% endset %}
+
+{% set addProxyRulesHtml %}
+  To add proxy rules,
+  follow the <a class="app-link app-link--underline app-link--text-colour" href="{{ proxyDocumentationUrl }}" target="_blank"
+     rel="noopener noreferrer">proxy documentation</a>.
+{% endset %}
+
+{% set serviceProxyTableRows = [] %}
+{% for domain in allowedDomains %}
+  {{ serviceProxyTableRows.push([
+    { html: appTag({
+      text: domain,
+      classes: "govuk-tag--turquoise",
```

Comment:
Not sure I'm a fane of this colour. How is `govuk-tag--purple` looking? If thats no good lets stick with the pattern we have on secrets and use `govuk-tag--green`